LiskHQ/lisk-sdk
# Add Accounts asset data to the API - Closes #3847

### What was the problem?

Asset field was not being returned when using http://localhost:4000/api/accounts

### How did I fix it?

By using extended option in the call to `storage.get` and adding the `asset` property to `accountFormater`

### How to test it?

- Build should be green
- `GET http://localhost:4000/api/accounts`
- Update an account with JSON for asset and `GET http://localhost:4000/api/accounts` contents of result for the updated account should match

### Review checklist

* The PR resolves #3847
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
